internal/controller/management_controller.go

```diff
@@ -142,13 +142,19 @@ func (r *ManagementReconciler) Update(ctx context.Context, management *hmc.Manag
 			continue
 		}
 
-		if _, _, err := helm.ReconcileHelmRelease(ctx, r.Client, component.helmReleaseName, r.SystemNamespace, helm.ReconcileHelmReleaseOpts{
+		hrReconcileOpts := helm.ReconcileHelmReleaseOpts{
 			Values:          component.Config,
 			ChartRef:        template.Status.ChartRef,
 			DependsOn:       component.dependsOn,
 			TargetNamespace: component.targetNamespace,
 			CreateNamespace: component.createNamespace,
-		}); err != nil {
+		}
+		reconcileInterval := template.Spec.Helm.Chart.Interval.Duration
+		if reconcileInterval != 0 {
+			hrReconcileOpts.ReconcileInterval = &reconcileInterval
+		}
```

Comment:
don't we want to set the default reconciliation interval here? as an analog to the `release` ctrl

Reply:
The default reconciliation interval is set by the admission mutation webhook (see changes in [internal/webhook/template_webhook.go](https://github.com/Mirantis/hmc/pull/699/files#diff-d026968b705a734fd88db7330857edc6977860ce0f9e044a203d4fc220d38c2e)). This field is required in the `HelmChartSpec` struct, it will always be populated, so I don't think we should set the defaults in the controller as well.